Rakefile

```
@@ -40,7 +40,7 @@ task :build => :environment do
   Rake::FileList["#{BUILD_DIR}/html/**/*.html"].each do |filename|
     File.open(filename, "r+") do |file|
       old_content = file.read
-      new_content = old_content.gsub(%r{<a href="(https?://(?!(?:www\.)?talkable\.com).*?)"}, '<a rel="nofollow" href="\1"')
+      new_content = old_content.gsub(%r{<a(.*)?href="(https?://(?!(?:www\.)?talkable\.com).*?)"}, '<a\1rel="nofollow" href="\2"')
```

There will always be at least one character between `<a` and `href`.
```suggestion
      new_content = old_content.gsub(%r{<a(.+)href="(https?://(?!(?:www\.)?talkable\.com).*?)"}, '<a\1rel="nofollow" href="\2"')
```